pkg/secrets/k8s/k8s_secrets_test.go

```diff
@@ -41,6 +41,26 @@ func TestKubernetesSecrets(t *testing.T) {
 			})
 		})
 
+		Convey("Given a map of data entries and a secret with backslashes", func() {
+			m := make(map[string][]byte)
+			// need to simulate a conjur secret, with 1 backslash, so we use unicode to simulate this
+			m["user"] = []byte("super\u005csecret");
```

Comment:
Maybe we can also test something like:    "\\\\\\\\a\u005\\\\b\\\c\\\\\"  
To verify algorithm doesn't fail on "corner cases"

Reply:
just ran it now and we get: \\a\u005\b\c\
which is a good sign! adding this test now (: